lib/github/commands/rest2html

```diff
@@ -102,9 +114,15 @@ class GitHubHTMLTranslator(HTMLTranslator):
             language = classes[1]
             del classes[:]
             self.body.append(self.starttag(node, 'pre', lang=language))
+        elif default_highlight_language is not None:
+            self.body.append(self.starttag(node, 'pre', lang=default_highlight_language))
         else:
             self.body.append(self.starttag(node, 'pre'))
 
+    def visit_highlight(self, node):
```

Comment:
I have no idea about rST, so forgive me if I'm missing the mark, but should this definition be here? It looks like we don't output to stdout at any other time except when returning from `main()`.